### ISSUE TITLE
makeabilitylab-test server logging not working

### ISSUE THREAD

**Issue by jonfroehlich** [jonfroehlich/makeabilitylabwebsite#869]
This is possibly related to https://github.com/jonfroehlich/makeabilitylabwebsite/issues/868

When I ssh'd and tried to look at the makeabilitylab-test logs, I noticed that the last log dates were from 2019. So, it looks like we're not properly logging on the test server anymore...

![image](https://user-images.githubusercontent.com/1621749/86498293-14075880-bd3a-11ea-9382-bcdcf787091d.png)

**Comment by mechanicjay** [comment]
This is an artifact of moving everything into the docker-compose file as the last iteration.  Previously I was keeping separate configs for test/prod which would do the appropriate volume mount to www-test or www as appropriate.

I haven't though through how to fix this at the moment, but I did manually override the config on test, so you can see the files....This will get overwritten at the next push.

**Comment by jonfroehlich** [comment]
Gotcha. Thanks. I was able to grab the new log file and figure out at least one cause of the error, which I just tried to fix and pushed a change (which will obviously now overwrite your manual configuration changes). So, if the test server does not come back up, I'll need you to make that same manual override again.

**Update**: looks like it worked. Wee!

But, I guess, longer term, we'll need a way to fix this so that I can consistently see the test server log files (as I obviously use makeabilitylab-test as a final check before deploying to production). Any thoughts on doing this that won't take too much of your effort/time?
